crates/sui-types/src/storage.rs

```diff
@@ -358,6 +372,7 @@ pub struct InMemoryStore {
     highest_verified_checkpoint: Option<(CheckpointSequenceNumber, CheckpointDigest)>,
     highest_synced_checkpoint: Option<(CheckpointSequenceNumber, CheckpointDigest)>,
     checkpoints: HashMap<CheckpointDigest, VerifiedCheckpoint>,
+    full_checkpoint_contents: HashMap<CheckpointSequenceNumber, FullCheckpointContents>,
```

Comment:
where is this written to? is it missing or can I just not find it

Reply:
line 590 in this file

Reply:
That's where it's read, but where is it written?